docs/docs/guide/platform/halo.md

```diff
@@ -6,36 +6,70 @@ order: 2
 
 HALO is a system of components for implementing decentralized identity designed around privacy, security, and collaboration requirements.
 
-*   The HALO [**application**](https://halo.dxos.org) is a software wallet that contains user identities, secrets (i.e.: private keys), other credentials.
-*   The HALO [**SDK**](https://www.npmjs.com/package/@dxos/client) is part of the DXOS client library and provides user authentication, identity, and contact management capabilities.
-*   The HALO **protocol** supports the verification, transport, and exchange of identity information between networked peers.
+- The HALO **Shell** manages the user's identity and connected devices. It's available as a [component](https://github.com/dxos/dxos/tree/main/packages/sdk/shell) and a [stand-alone application](https://halo.dxos.org).
+- The HALO [**SDK**](https://www.npmjs.com/package/@dxos/client) is part of the DXOS client library and provides user authentication, identity, and contact management capabilities.
+- The HALO **protocol** supports the verification, transport, and exchange of identity information between networked peers.
 
-The HALO application also acts as a secure vault where all ECHO data is physically stored on behalf of the user and their applications. Read more about [local vault topology](../platform/#local-vault-topology).
+The HALO application also acts as a secure vault where all ECHO data is physically stored on behalf of the user and their applications.
```

Comment:
as of this release the halo app is effectively deprecated, it won't have any idea who you are unless you invite it to your identity (but there's no reason to do that because it doesn't provide any extra functionality)

Reply:
it has the "additional functionality" of giving you access to the Shell _without_ Composer. For me at least, that has value since Composer dies on me semi-regularly. With the Halo app, I can invite another device, manage my agent, etc.

Reply:
we're gonna leave the reference to the stand-alone application, for now, since halo.dxos.org will still be deployed.